src/Directory.Build.props

```diff
@@ -3,12 +3,18 @@
   <Import Project="../Directory.Build.props" />
 
   <PropertyGroup Label="Common package properties">
+    <Company>Patrik Hägne</Company>
     <Copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</Copyright>
     <PackageProjectUrl>https://fakeiteasy.github.io/</PackageProjectUrl>
     <PackageIconUrl>https://fakeiteasy.github.io/img/fakeiteasy_logo_256_square_white.png</PackageIconUrl>
     <PackageLicenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</PackageLicenseUrl>
     <PackageReleaseNotes>https://github.com/FakeItEasy/FakeItEasy/releases</PackageReleaseNotes>
     <NoPackageAnalysis>true</NoPackageAnalysis>
+    <MinVerMajorMinor>5.0</MinVerMajorMinor>

```

Comment:
Indeed. I would prefer not to have to update this ever. Truth be told, I'd rather it not exist at all.

Reply:
Is this something that's only necessary until we stop using GitFlow? That is, once we're developing on master where there's a recent ancestor with (for example) a 5.0.0 tag, it'll all work out without this setting?
And any tags we explicitly add for pre-release versions would be honoured?

Reply:
This is all detailed in https://github.com/adamralph/minver#can-i-bump-the-major-or-minor-version.

> If we don't update this, will it still honor the tags? e.g. when we release 5.1.0, do we need to update MinVerMajorMinor, or is it enough to create a 5.1.0 tag ?

Yes, it's enough to just create the tag.

> Indeed. I would prefer not to have to update this ever. Truth be told, I'd rather it not exist at all.

It's not required. All it does is bump interim builds before the first `5.0.x` tag to `5.0.0-alpha.0.{height}`. If you prefer, I'll remove it.

> Is this something that's only necessary until we stop using GitFlow?

It has nothing to do with your branching strategy. E.g. if you switched to GitHub Flow or Release Flow, there will still be a time when you decide "`master` is now 5.1.0". If you don't have this property, MinVer will keep calculating `5.0.x` version numbers until the first `5.1.x` tag.

> And any tags we explicitly add for pre-release versions would be honoured?

Absolutely. All this property does is to ensure that the version is no lower than the specified `MAJOR.MINOR`. At one time during the alpha phase, it was named `MinVerMinimumMajorMinor` which made that more explicit, but I found the name unwieldy so I changed it do just `MinVerMajorMinor`. If the longer name would help to avoid question, I guess we could change it back, but my hope is that the FAQ I linked above is enough.